Libraries/WebSocket/RCTSRWebSocket.m

```diff
@@ -1345,94 +1341,138 @@ - (void)stream:(NSStream *)aStream handleEvent:(NSStreamEvent)eventCode;
       }
     }
   }
-
-  assert(_workQueue != NULL);
+  
+  // _workQueue cannot be NULL
+  if (!_workQueue) { return; }
```

Comment:
If work queue cannot be null can we just remove this?

Reply:
@janicduplessis In theory, we don't need to check work queue nullable, the reason I add this is to add the further safe check, because if work queue is null, it would crash immediately.